### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Get Shared Access Signature_Shared Access Signature_ScreenReader:NVDA announcing "show date and time picker" option as "sub menu"

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5707]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_Azure Tools Storage Explorer_May2022; #Azure Tools Storage Explorer; #WCAG1.3.1; #ScreenReader; #NVDA; #FTP; #CEACCApproved; #A11yCAIOH; #Closed;

## Actual results:
NVDA announcing 'show date and time picker' option present in the "start time" and "Expiry time" as " show date and time submenu".

## Expected results:
NVDA shouldn't announce 'show date and time picker' option as "menu item". It should be only announced as button.

## Environmental Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Open Explorer" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Right click on blob and select "Get Shared Access Signature" control and hit enter.
6. Turn on NVDA.
7. Verify 'show date and time picker' option present in the "Start time" and "expire time" field are MAS compliant.

## User Impact:
Users will be confused if something that isn't a "menu item" are announced as "menu item".

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Attachments

https://user-images.githubusercontent.com/105198219/168733349-58aa9ca4-33f0-44ce-a59f-90509fb65813.mp4

**Comment by MRayermannMSFT** [comment]
Went to accessibility office hours. Both of these should end being closed as not-a-bug/won't fix.

**Comment by ManishaC1** [comment]
Closing the bug.
